helm-chart/kuberay-operator/templates/ray_rayjob_viewer_role.yaml

```diff
@@ -1,24 +1,19 @@
-# permissions for end users to view rayjobs.
+# permissions for end users to edit rayjobs.
 {{- if and .Values.rbacEnable (not .Values.singleNamespaceInstall) }}
-
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  labels: {{ include "kuberay-operator.labels" . | nindent 4 }}
   name: rayjob-viewer-role
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
 rules:
 - apiGroups:
   - ray.io
   resources:
   - rayjobs
+  - rayjobs/status
```

Comment:
Why to grant additional `list` and `watch` permission for `rayjobs/status`?

Reply:
I just refer to kubernetes default clusterrole system:aggregate-to-view https://github.com/kubernetes/kubernetes/blob/b4c6895d0b0a913e3461bdc78358aa9514604b8f/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L111, it grant */status to view clusterrole. If rayxxx/status is not appropriate to be here, I can remove it.

Reply:
cc @kevin85421 Do you think it is okay to grant those additional permissions to status? Personally, I think it is fine to grant them.

Reply:
It's fine with me. I plan to revisit all RBAC permissions soon and can decide whether to remove it at that time. We can leave it as is for this PR.